backend/testsuite/common.go

```diff
@@ -0,0 +1,63 @@
+package testsuite
+
+import (
+	"os"
+
+	"github.com/c2fo/vfs/v6"
+	"github.com/c2fo/vfs/v6/backend/azure"
+	"github.com/c2fo/vfs/v6/backend/ftp"
+	"github.com/c2fo/vfs/v6/backend/gs"
+	"github.com/c2fo/vfs/v6/backend/mem"
+	_os "github.com/c2fo/vfs/v6/backend/os"
+	"github.com/c2fo/vfs/v6/backend/s3"
+	"github.com/c2fo/vfs/v6/backend/sftp"
+)
+
+func CopyOsLocation(loc vfs.Location) vfs.Location {
```

Comment:
Doesn't look like our PR checks prevent merging on lint checks, but these are are missing the comments.

Reply:
I think because they're in tests.  golangci is a collection of many linters.  I'll bet the one that deals with comments (golint/revive) skips for tests.

Reply:
I think because they're in tests.  golangci is a collection of many linters.  I'll bet the one that deals with comments (golint/revive) skips for tests.
It passes golangci-lint locally and in CI.  I did have to fix some other lint issues earlier. 
https://github.com/C2FO/vfs/blob/master/.golangci.yml#L215